pandas/tests/frame/methods/test_fillna.py

```diff
@@ -647,3 +647,87 @@ def test_fillna_nonconsolidated_frame():
     df_nonconsol = df.pivot("i1", "i2")
     result = df_nonconsol.fillna(0)
     assert result.isna().sum().sum() == 0
+
+
+def test_fillna_with_int_and_string_nonempty():
+    df = DataFrame(
+        {
+            "A": [1, 2, np.nan],
+            "B": [4, np.nan, 8],
+        },
+        dtype="Int64",
+    )
+    df.fillna("nan")
+
+    expected = df = DataFrame(
```

Comment:
probably don't want to set both `df` and `expected` in this line?

Reply:
same in each case below

Reply:
Interesting, I don't remember setting both df and expected like that - might have been a product of running pre-commit or something else? I can try to change this asap.

Reply:
can't imagine the pre-commit tooling doing this.  most likely just a typo.  happens to us all.